codelists/+partyRole.csv

```diff
@@ -5,3 +5,11 @@ processContactPoint,Process contact point,A contact point dedicated to this cont
 reviewContactPoint,Review contact point,The service from which information about the review procedure can be obtained.
 selectedParticipant,Selected participant,A party that has already been selected to participate in the design contest.
 informationService,Information service,An organization that can provide information regarding specific aspects of the contracting process.
+procurementServiceProvider,Procurement service provider,An organization that provides procurement services to contracting authorities.
+eSender,TED eSender,An organization that provides data transfer services to contracting authorities.
+leadBuyer,Lead buyer,The lead buyer in a group of buyers.
+awardingCentralPurchasingBody,Awarding central purchasing body,"A central purchasing body awarding public contracts or concluding framework agreements for goods, services or works intended for other buyers."
```

Comment:
The profile doesn't use the code 'procuringEntity' as far as I can tell. Let's just map to that code instead of creating this code.

Reply:
Sounds good. I've updated `guidance.yaml` and removed the new code in https://github.com/open-contracting-extensions/ocds_eu_extension/pull/12/commits/590075767965d824effbc8eb94f031fe41dcd665